php/password.php

```diff
@@ -0,0 +1,46 @@
+<?php
+    // Start a new PHP session (or continue an existing one)
+    session_start();
+    $pwhash =  parse_ini_file("/etc/pihole/setupVars.conf")['WEBPASSWORD'];
+
+    // If the user wants to log out, we free all session variables currently registered
+    if(isset($_GET["logout"]))
+        session_unset();
+
+    // Test if password is set
+    if(strlen($pwhash) > 0)
```

Comment:
No. It is unlikely, but it might be that someone deletes the password while he is logged in. Then strange thinks would happen if he clicks on "Logout". If the user visited "index.php?logout" I still have to check all the various possibilities.